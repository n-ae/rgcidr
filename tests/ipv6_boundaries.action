::/0 tests/ipv6_formats.given
