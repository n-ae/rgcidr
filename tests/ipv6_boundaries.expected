2001:db8::1
2001:0db8:0000:0000:0000:0000:0000:0001
2001:db8:85a3::8a2e:370:7334
2001:db8:85a3:0:0:8a2e:370:7334
2001:db8::
::1
::
2001:0db8:85a3:0000:0000:8a2e:0370:7334
2001:db8:1234:5678:90ab:cdef:1234:5678
2001:db8::192.168.1.1
fe80::1
2002:db8::1
